golang/pkg/cli/config_file.go

```diff
@@ -24,21 +25,27 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-// Settings and state of the application
-type Settings struct {
-	SettingsFile        SettingsFile
+// Though the CLI itself is stateless
```

Comment:
Personal comment? Or create a task for this.

Reply:
It is just the ironic a bit, state for a cli application exists per-execution, i'm going to explain this aiding the code readers' mental model.

Reply:
OK